aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -485,6 +485,7 @@ pub enum EntryFunctionCall {
         num_signatures_required: u64,
         metadata_keys: Vec<Vec<u8>>,
         metadata_values: Vec<Vec<u8>>,
+        timeout_duration: u64,
```

Comment:
It would be nice addition if the usage discription is also included for `timeout_duration`

Reply:
@so-kkroy22 this part of the code is auto-generated. The description is present in the `multisig_account.move` for `timeout_duration` parameter of the account.